docs/use-service.md

```diff
@@ -17,68 +17,85 @@ Once you have your Mapzen API key you'll need include it with Terrain Tile reque
 Request a single tile with this URL pattern to get started:
 
 ```
-https://tile.mapzen.com/mapzen/terrain/v1/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
+https://tile.mapzen.com/mapzen/terrain/v2/{format}/{z}/{x}/{y}.{extension}?api_key=your-mapzen-api-key
```

Comment:
I agree. We might want separate buckets internally so that we can roll back. However, if the outputs of this version are fully compatible with the existing `v1`, but with much more detailed data in many places, then I think everyone using `v1` would appreciate the "automagical" upgrade.